docs/how-to/rocks/index.rst

```diff
@@ -8,6 +8,5 @@
    Publish a rock to a registry <publish-a-rock>
    Migrate a Docker image to a chiselled rock <migrate-to-chiselled-rock>
    Use the flask extension <use-flask-extension>
-   Use the django extension <use-django-extension>
```

Comment:
Why is django guide removed?

Reply:
My request. These two extensions need a lot of work, so it's safer to remove this for now as it could lead people to the same ill-documented guides we're trying to fix for Flask.

The same work that is being done for flask must be done for Django, so as soon we we're done with this, a similar PR should be raised to revive the django docs